tests/integration/transformers/test_categorical.py

```diff
@@ -198,6 +198,28 @@ def test_frequency_encoder_numerical_nans():
     pd.testing.assert_frame_equal(reverse, data)
 
 
+def test_frequency_encoder_numerical_nans_no_warning():
+    """Ensure FrequencyEncoder does not emit FutureWarning with nan values.
+
+    Related to Issue #793 (https://github.com/sdv-dev/RDT/issues/793)
+    """
+    # Setup
+    data = pd.DataFrame({
+        'column_name': pd.Series([1, 2, float('nan'), np.nan], dtype='object')
+    })
+    column = 'column_name'
+
+    # Run and Assert
+    transformer = FrequencyEncoder()
+    with warnings.catch_warnings():
+        warnings.simplefilter('error', FutureWarning)
+        transformer.fit(data, column)
+        transformed = transformer.transform(data)
+        reverse = transformer.reverse_transform(transformed)
+
```

Comment:
```suggestion
    with pytest.warns(record=True) as record:
        transformer.fit(data, column)
        transformed = transformer.transform(data)
        reverse = transformer.reverse_transform(transformed)
    # look in records to make sure no FutureWarning raised?
```

Reply:
Oh, I see. Nvm you did it right.
https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#additional-use-cases-of-warnings-in-tests